lib/moped/sockets/tcp.rb

```diff
@@ -14,9 +14,9 @@ class TCP < ::TCPSocket
       # @param [ Integer ] port The port.
       #
       # @since 1.2.0
-      def initialize(host, port)
+      def initialize(host, port, options)
```

Comment:
Little weird that the options is added here, but then unused -- should it be made optional/empty by default?

Reply:
Also, should they be renamed to `ssl_options`?